config/bootstrap.sh

```diff
@@ -14,12 +14,13 @@ apt-get update
 install sqlite3
 
 # install python
-install python
-install python-pip
-install python-dev
-install python-software-properties
+install python3
+install python3-pip
+install python3-dev
+install python3-software-properties
+install git
 
```

Comment:
Add the following lines here:
wget https://raw.github.com/pypa/pip/master/contrib/get-pip.py
sudo python3 get-pip.py
rm get-pip.py